cloudigrade/config/urls.py

```diff
@@ -18,6 +19,8 @@
 router.register(r'image', MachineImageViewSet)
 router.register(r'report', ReportViewSet, base_name='report')
 router.register(r'sysconfig', SysconfigViewSet, base_name='sysconfig')
+router.register(r'report/accounts', CloudAccountOverviewViewSet,
+                base_name='overview')
```

Comment:
Although not required, the convention for naming routes is to match the relevant object's class name (if it's a typical CRUD endpoint) or roughly match the URL path it's registered to. That makes it more obvious when debugging or if you need to manually `reverse` a URL later. For something like this, a name like `report-accounts` might be appropriate.